### PR TITLE
move existing modal component to holocene

### PR DESCRIPTION
**Pull request by rossedfort**
## What was changed

- moves the existing Modal component to Holocene and adds a Chapter in Fiction
- Also makes menu and menu-item styles consistent across selects/dropdown-buttons/split-buttons.
- Fixes a bug with indeterminate checkboxes
- Fixes a typo in file naming (primatives -> primitives)
- Disambiguate Cards and Alerts

### Modal
Screen Shot 2022-07-27 at 9 31 43 AM

### Card
Screen Shot 2022-07-27 at 9 30 31 AM

### Alert
Screen Shot 2022-07-27 at 9 29 37 AM

## Why?

- need a modal in cloud-ui
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Review by stevekinney — APPROVED** [review]

File: src/lib/holocene/select/option.svelte [hunk 1]
```diff
@@ -18,7 +18,7 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
   import Icon from '$holocene/icon/index.svelte';
-  import MenuItem from '$holocene/primatives/menu/menu-item.svelte';
+  import MenuItem from '$holocene/primitives/menu/menu-item.svelte';
```
Comment (GiantRobots) [inline src/lib/holocene/select/option.svelte:21]: I prefer primat ives 🦍

File: src/lib/holocene/split-button.svelte [hunk 2]
```diff
@@ -1,47 +1,56 @@
...
-    @apply cursor-not-allowed opacity-50;
-  }
+    :global(.segment) {
+      @apply relative flex w-fit items-center justify-center py-2 font-secondary text-sm;
+    }
 
-  :global(.split-button-menu) {
-    @apply flex min-w-max flex-col items-start gap-y-4 border-gray-300 p-3 px-6 font-secondary text-sm;
-  }
+    :global(.segment.left) {
```
Comment (GiantRobots) [inline src/lib/holocene/split-button.svelte:40]: I was thinking on these you could do the terrible thing of :first-child and :last-child and then the css becomes automagic for any amount of containing elements without having to ho hum about what is the leftmost part of the button and the right most. This is totally fine too just a thought

Comment (rossedfort) [reply src/lib/holocene/split-button.svelte:40]: Ah yeah, I think for split button this should be OK, but for something like the Toggle Buttons, we will definitely want to do that.
Screen Shot 2022-07-27 at 10 37 41 AM

Comment (GiantRobots) [reply src/lib/holocene/split-button.svelte:40]: Yeah when I think about it those feel like they could/should be composed of the more primative 🦍 elements. But this approach is also fine for right meow

**Review by GiantRobots — APPROVED** [review]

**Pull request merged**